.github/no-response.yml

```diff
@@ -12,5 +12,5 @@ closeComment: >-
   resolve this issue. We are therefore reluctantly going to close this
   bug for now. Please don't hesitate to comment on the bug if you have
   any more information for us; we will reopen it right away!
-  
+
```

Comment:
Are we sure this doesn't break github's parsing of this file?

Reply:
no, not really. we'll find out pretty quick though. :-)